src/Builder/OpBuildTable.inc

```diff
@@ -210,7 +210,7 @@ op_dialect_version_map_["TreeEnsembleClassifier"] = {1};
 op_dialect_version_map_["TreeEnsembleRegressor"] = {1};
 op_dialect_version_map_["Unique"] = {11};
 op_dialect_version_map_["Unsqueeze"] = {13, 11};
-op_dialect_version_map_["Upsample"] = {9, 7};
+op_dialect_version_map_["Upsample"] = {10, 7};
```

Comment:
It looks like we have to keep 9 in the set, e.g. {10, 9, 7}. And defining a simple rule in `decompose.cpp` for `UpsampleV9Op`.